ssl/quic/quic_channel.c

```diff
@@ -1238,36 +1237,6 @@ static int ch_on_transport_params(const unsigned char *params,
             got_orig_dcid = 1;
             break;
 
-        case QUIC_TPARAM_RETRY_SCID:
-            if (ch->is_server) {
-                reason = TP_REASON_SERVER_ONLY("RETRY_SCID");
-                goto malformed;
-            }
-
-            if (got_retry_scid) {
-                reason = TP_REASON_DUP("RETRY_SCID");
-                goto malformed;
-            }
-
-            if (!ch->doing_retry) {
-                reason = TP_REASON_NOT_RETRY("RETRY_SCID");
-                goto malformed;
-            }
-
-            if (!ossl_quic_wire_decode_transport_param_cid(&pkt, NULL, &cid)) {
-                reason = TP_REASON_MALFORMED("RETRY_SCID");
-                goto malformed;
-            }
-
-            /* Must match Retry packet SCID. */
-            if (!ossl_quic_conn_id_eq(&ch->retry_scid, &cid)) {
-                reason = TP_REASON_EXPECTED_VALUE("RETRY_SCID");
-                goto malformed;
-            }
-
-            got_retry_scid = 1;
-            break;
-
```

Comment:
Why did you remove this? This does not look right. Please see https://www.ietf.org/rfc/rfc9000.html#cid-auth

Reply:
I agree, this removal makes PR too invasive. I prefer to keep it.